packages/ui/src/components/vuestic-mixins/FormComponent/cFormComponent.ts

```diff
@@ -0,0 +1,170 @@
+import { ref, toRefs, computed, inject, onMounted, onUnmounted } from 'vue'
+import { FormProvider, FormServiceKey, FormElement } from '../../vuestic-components/va-form/consts'
+
+const prepareValidations = (messages: string | any[] = [], callArguments = null) => {
+  if (typeof messages === 'string') {
+    return [messages]
+  }
+  return messages
+    .map((message) => typeof message === 'function' ? message(callArguments) : message)
+}
+
+export function useFormProvider (context: FormElement) {
+  const formProvider: FormProvider | undefined = inject(FormServiceKey, undefined)
```

Comment:
You can early return instead of conditional check later.
I.e.
```ts
if (!formProvider) return;
```
As `formProvider` is not reactive either way.